### ISSUE TITLE
[❔ other question] BumpMap not working with Inverse Rendering

### ISSUE THREAD

**Issue by priyamvad** [mitsuba-renderer/mitsuba2#297]
## Summary
I get the following error when only trying to inverse render the BumpMap texture:
_RuntimeError: set_gradient(): no gradients are associated with this variable (a prior call to requires_gradient() is required.)_

I know vertex buffers were not able to be inverse rendered based on a prior issue. Is there a similar problem with BumpMaps as well? I see NormalMap BSDF does not have a traverse function. If I add it to display the normal map texture, would I not be able to Inverse render the normal map either?

Thank you for looking into it!

## Steps to reproduce
Only keep the bump map texture as the parameter to inverse render.
